bin/assigner/src/main.cpp

```diff
@@ -520,6 +671,21 @@ int curve_dependent_main(std::string bytecode_file_name,
             crypto3::zk::snark::plonk_constraint_system<BlueprintFieldType>;
     using AssignmentTableType = zk::snark::plonk_table<BlueprintFieldType, zk::snark::plonk_column<BlueprintFieldType>>;
 
+    using Endianness = nil::marshalling::option::big_endian;
+    using TTypeBase = nil::marshalling::field_type<Endianness>;
+    using constant_column_marshalling_type =
```

Comment:
Better to define marshaling types in the one place: `crypto3/libs/marshalling/zk/include/nil/crypto3/marshalling/zk/types/plonk/assignment_table.hpp`